webui/react/src/services/decoder.ts

```diff
@@ -613,7 +616,28 @@ const decodeSummaryMetrics = (data: unknown): types.SummaryMetrics => {
   };
 };
 
-export const decodeV1TrialToTrialItem = (data: Sdk.Trialv1Trial): types.TrialItem => {
+const decodeSummaryMetricsValue = (
```

Comment:
consider naming this such that it's apparent that we're getting the validation metrics from the summary metrics? Maybe something like `decodeSummaryValidationMetrics`?